WordPress/Classes/ViewRelated/Blog/BlogPersonalization/BlogDashboardPersonalizationViewModel.swift

```diff
@@ -1,9 +1,15 @@
 import SwiftUI
 
 final class BlogDashboardPersonalizationViewModel: ObservableObject {
+    let quickActions: [DashboardPersonalizationQuickActionViewModel]
     let cards: [BlogDashboardPersonalizationCardCellViewModel]
 
-    init(service: BlogDashboardPersonalizationService, quickStartType: QuickStartType) {
+    init(blog: Blog, service: BlogDashboardPersonalizationService, quickStartType: QuickStartType) {
```

Comment:
FYI `BlogDashboardPersonalizationViewModelTests` is failing now with the error: `missing argument for parameter 'blog'`

Reply:
Oops, forgot I wrote these – updated.